Index: /etc/dhcpcd.conf
===================================================================
--- /etc/dhcpcd.conf
+++ /etc/dhcpcd.conf
@@ -57,3 +57,15 @@ slaac private
 # fallback to static profile on eth0
 #interface eth0
 #fallback static_eth0
+
+# [pi-topOS] START
+#
+# Set static IP for USB-OTG virtual Ethernet interface
+#
+# Notes:
+# DHCP is configured to offer IP addresses via USB-OTG virtual Ethernet in /etc/default/isc-dhcp-server
+# Compatible subnet, netmask and IP ranges are specified in /etc/dhcp/dhcpd.conf
+interface ptusb0
+static ip_address=192.168.64.1/24
+#
+# [pi-topOS] END
